protoV2/README.md

```
@@ -75,23 +75,26 @@ npm run generate:skip-clone  # Quick regeneration
 ### 4. Build
 
 ```bash
-npm run build  # Builds to proto-ts/esm/
+pnpm run build  # Builds to proto-ts/
 ```
 
 ## Directory Structure
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)